api_types.ts

```diff
@@ -984,8 +1005,7 @@ export interface ExecutionContext {
   /**
    * Information about state of the current sync. Only populated if this is a sync table formula.
    */
-  readonly sync?: Sync<unknown, unknown>;
-
+  readonly sync?: Sync<unknown>;
```

Comment:
your plan is for this to be present for all metadata functions invoked in the context of a sync, right? I was planning to move `executionId` into Sync, assuming this would be available in more than just the sync formula itself.

Reply:
Yeah, exactly. I was thinking we only keep it on `SyncExecutionContext` but it's hard to model unless we have special `makeParameter` functions that work only within sync tables and pass the context through. For now, just keeping as is.